### PR TITLE
Added tooltips for inspector, settings, and search controls

### LINKED ISSUE #772: "Click-to-select" and "Theme selector" needs visual feedback e.g. tooltips
I'm scared to click them before I hover and know what they will do. :)

### PR DESCRIPTION
**Pull request by bvaughn**
Resolves #772

screen shot 2017-06-01 at 12 48 24 pm

screen shot 2017-06-01 at 12 48 29 pm
screen shot 2017-06-01 at 12 50 07 pm

### CODE REVIEW

File: frontend/SettingsPane.js [hunk 1]
```diff
@@ -136,8 +136,9 @@ class SettingsPane extends React.Component {
             onFocus={() => this.setState({focused: true})}
             onBlur={() => this.setState({focused: false})}
             onKeyDown={e => this.onKeyDown(e.key)}
-            placeholder={this.props.placeholderText}
+            placeholder="Search (text or /regex/)"
             onChange={e => this.props.onChangeSearch(e.target.value)}
+            title="Search React elements"
```
Comment (gaearon) [inline frontend/SettingsPane.js]: I would suggest "Search React component names and text"

File: frontend/SettingsPane.js [hunk 2]
```diff
@@ -185,6 +184,7 @@ const InspectMenuButton = ({ isInspectEnabled, onClick, theme }) => (
   <button
     onClick={onClick}
     style={inspectMenuButtonStyle(isInspectEnabled, theme)}
+    title="Select an element in the page to inspect it"
```
Comment (gaearon) [inline frontend/SettingsPane.js]: I would suggest adding "React" in there. (Since it doesn't work otherwise)

File: frontend/SettingsPane.js [hunk 3]
```diff
@@ -196,7 +196,11 @@ const InspectMenuButton = ({ isInspectEnabled, onClick, theme }) => (
 );
 
 const SettingsMenuButton = ({ onClick, theme }) => (
-  <button onClick={onClick} style={styles.settingsMenuButton}>
+  <button
+    onClick={onClick}
+    style={styles.settingsMenuButton}
+    title="Customize devtools"
```
Comment (gaearon) [inline frontend/SettingsPane.js]: Maybe "Customize React DevTools". I don't think we use "devtools" capitalization anywhere except package name.

**Comment by bvaughn** [comment]
🤣  such a small pr, so many requested changes

**Pull request merged**